projects/lf-documentation/src/index.html

```diff
@@ -6,7 +6,8 @@
   <title>LfDocumentation</title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="icon" type="image/x-icon" href="favicon.ico" />
-  <link href="https://lfxstatic.com/npm/@laserfiche/lf-ui-components@14/cdn/indigo-pink.css" rel="stylesheet"/>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block" rel="stylesheet">
```

Comment:
is there a reason for this change?
Assuming the indigo-pink moved into styles.css, and the material link was automatically added?